sphinx_ansible_theme/extrafooter.html

```diff
@@ -1,4 +1,4 @@
-<!-- extra footer elements for Ansible beyond RTD Sphinx Theme --->
+<!-- extra footer elements for Ansible beyond RTD Sphinx Theme -->
```

Comment:
If anyone wonders why this is wrong: see https://www.w3.org/html/wg/spec/syntax.html#comments, "[...] nor end with a U+002D HYPHEN-MINUS character (-)."